hack/versions-management/github_auth.sh

```diff
@@ -32,6 +32,8 @@ if [ -z "$INSTALLATION_TOKEN" ]; then
 fi
 
 export GITHUB_APP_PRIVATE_KEY=$(cat ${GITHUB_APP_PRIVATE_KEY_PATH})
-git config --global url."https://x-access-token:${INSTALLATION_TOKEN}@github.com/".insteadOf "https://github.com/"
+git remote add origin "https://x-access-token:${INSTALLATION_TOKEN}@github.com/openshift-assisted/cluster-api-provider-openshift-assisted.git" 2>/dev/null || true
```

Comment:
Since we're here, how about:
* make remote configurable (`$REMOTE`)
* by default a name that doesn't normally exist (normally origin and upstream are quite popular)

This way we can easily work with this locally for debugging/development